docs/UserGuide.md

```diff
@@ -253,29 +255,35 @@ Example:
 
 ### Reschedules an appointment for a patient : `change`
 
-Reschedules or modifies an existing appointment with a new date, time and duration.
+Reschedules or modifies an existing appointment with a new date, time or duration.
 
-Format: `change APPT_INDEX d/DATE t/TIME dur/DURATION`
+Do note that the duration used here is **measured in minutes**!
+
+Format: `change APPT_INDEX [d/DATE] [t/TIME] [dur/DURATION]`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-Do note that the duration used here is measured in minutes!
+If you would only like to change the duration of an appointment while keeping it's original date and time, simply 
+enter the new `DURATION` with it's corresponding `APPT_INDEX`! There is no need to retype it's date and time. The same 
+applies for the other fields.
 </div>
 
-* Reschedules an appointment at the specified `APPT_INDEX` for the patient it is tagged to. 
-The index here refers to the index number shown on the left side of the displayed appointment list on the right side of the UI. 
+* **Reschedules an appointment** at the specified `APPT_INDEX` for the patient it is tagged to. 
+* The `APPT_INDEX` here refers to the number shown on the left side of the displayed appointment book located on the right side of the UI. 
 * Please note that the index used **must be a positive integer** 1, 2, 3, …
-* Please note that the `DATE` and `TIME` used for rescheduling must take place in the future.
+* Please note that the `DATE` and `TIME` used for rescheduling must be set in the future.
+* **At least one** of the optional fields must be present for a successful execution of the change command.
 
 * If you wish to **modify** the duration of an existing appointment, simply call the command in the following format:
-    * `change 1 d/ORIGINAL_DATE t/ORIGINAL_TIME dur/NEW_DURATION` Extends the duration of the appointment at `INDEX` 1 to the new `DURATION` with the same date and start time.
+    * `change 1 dur/NEW_DURATION`: <br>Extends the duration of the original appointment at `APPT_INDEX` 1 to the `NEW_DURATION` with the same date and start time.
```

Comment:
Would it be better if we refer to the original appointment as the first appointment in the appt list?